learn/how-tos/edit-datatable-bound-to-queryapi.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Configuring edit functionality for the datatable that is bound to Query API"
+id: ""
+sidebar_label: "Edit functionality for the datatable that is bound to Query API"
+---
+---
+
+When the datatable widget is bound to the query API, the records cannot be edited. Incase, you would like to add the edit funtionality, you can achive your requirement using this approach.
```

Comment:
Please mention why you cannot edit and what are the risks associated with it when you say "you cannot edit the data table bound to query API".